mattermost-10.2.advisories.yaml

```diff
@@ -579,6 +579,11 @@ advisories:
             componentType: apk
             componentLocation: /.PKGINFO
             scanner: grype
+      - timestamp: 2024-12-04T14:41:29Z
+        type: false-positive-determination
+        data:
+          type: component-vulnerability-mismatch
```

Comment:
This isn't a `component-vulnerability-mismatch` since the vulnerability describes `mattermost`, which is the same component that the scanner found in the artifact.

`vulnerable-code-version-not-used` would be a more precise fit here.